alsa_settings/PTLP_RVP_SDW.sh

```diff
@@ -0,0 +1,6 @@
+set -e
+
+# enable headset playback and capture
+amixer -c sofsoundwire cset name='Headset Mic Switch' on
+amixer -c sofsoundwire cset name='rt722 FU0F Capture Switch' on
+amixer -c sofsoundwire cset name='rt722 FU0F Capture Volume' 50
```

Comment:
Not sure if we need to set 'rt722 FU0F Capture Volume'. Maybe it is better to keep it as the default value?

Reply:
For the default value of 75, the test fails with the error message 'over amplitude detected.' We reduced the capture volume and tested values between 30 and 50, which worked fine. Hence, we kept the same value here